capabilities/index/index.ipldsch

```diff
@@ -0,0 +1,5 @@
+type V1Link link
+
+type AddCaveats struct {
+	index V1Link
+}
```

Comment:
I think this can just be:
```
type AddCaveats struct {
    index link
}
```
since `link` is a builtin type. Though `V1Link` is also already defined here: https://github.com/storacha/go-libstoracha/blob/main/capabilities/types/types.ipldsch#L26
Unsure what the significance of `V1Link` vs `link` is.

Reply:
I don't totally understand IPLD schema syntax, but I had the same thought and switching it seemed to allow V0 CIDs, while leaving it only allowed V1. No clue what kind of magic makes that happen… 😕